travis-scripts/fontforge.rb

```diff
@@ -53,7 +49,6 @@ class Fontforge < Formula
   depends_on "pango"
   depends_on "zeromq"
   depends_on "czmq"
-  depends_on "fontconfig"
```

Comment:
It's still there. Moved under the `depends_on :python` line below. It's currently surrounded in bug links, so we moved it for clarity.